ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/transaction/Rejections.scala

```diff
@@ -52,8 +54,6 @@ private[transaction] class Rejections(metrics: Metrics) {
       transactionEntry: DamlTransactionEntrySummary,
       rejection: Rejection,
   )(implicit loggingContext: LoggingContext): DamlTransactionRejectionEntry.Builder = {
-    logger.trace(s"Transaction rejected, ${rejection.description}.")
```

Comment:
Do we log this somewhere else, e.g. in post-execution?

Reply:
In the function above.

Reply:
And also in post-execution I think, but I'll double-check.

Reply:
It looks like there are at least some post-executors that don't log which write set they've chosen => further PRs will come.

Reply:
I'll fix at least one in this PR though, as it seems fairly straightforward.

Reply:
524e1a7115ac43a9f89b6e404c3271bd5f867242